Description: Fix incorrect use of memset count/value (#609085)
 In ./matchbox-panel_0.9.3/src/panel.c
   memset(panel, sizeof(MBPanel), 0);
 This should be memset(panel, 0, sizeof(MBPanel))
Author: Silvio Cesare <silvio.cesare@gmail.com>
Last-Update: 2011-01-06
--- a/src/panel.c
+++ b/src/panel.c
@@ -1599,7 +1599,7 @@
   XSetErrorHandler(util_handle_xerror);
    
   panel = NEW(MBPanel);
-  memset(panel, sizeof(MBPanel), 0);
+  memset(panel, 0, sizeof(MBPanel));
 
   /* defualts */
 
